orders/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from orders.views import CartView
+
+urlpatterns = [
+    path('/cart', CartView.as_view())
```

Comment:
목록을 반환하는 엔드포인트이므로 cart -> carts로 수정해주세요.
```suggestion
    path('/carts', CartView.as_view())
```